### PR TITLE
Adds scrollbar and default label

### PR DESCRIPTION
**Pull request by colbydehart**
This adds a scrollbar and max-height to the dropdown options as well as defaults the label to the current selected item, if there is one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -629,7 +629,7 @@ class MyComponent extends React.Component {
 | `options[ <Object> ]`| `array`         | Array of object options to render the options in the dropdown                                  |
 | `option.label`       | `string`        | Actual label of the item to be toggled and rendered                                            |
 | `option.value`       | `any`           | Actual value to send to the `onChange` handler when an item is selected                        |
-| `label`              | `string`        | Default label when no item is selected                                                         |
+| `label`              | `string`        | Default label when no item is selected. Defaults to currently selected label or ''             |
```
Comment (sethen) [inline README.md]: Defaults to currently selected label or this value?

**Comment by sethen** [comment]
The description on the label is a bit weird.

**Comment by colbydehart** [comment]
How about `Label for the DropDown. Default is the label of the currently selected option`

**Comment by sethen** [comment]
That's fine.

**Comment by colbydehart** [comment]
Cool, fixed.

**Comment by sethen** [comment]
Squash

**Pull request merged**